login/token_context/token.go

```diff
@@ -0,0 +1,32 @@
+// Package token_context contains the code that extract token manager from the
+// context.
+package token_context
```

Comment:
That's right, it's not used. It's used only in tests which I think just outdated and should be rewritten or removed. I can help @hectorj2f with that tests but there is another problem. If we keep the context in the token package then we can't use log in the token package. We need to keep context in the top level so it can be used by another top level stuff like log. So, maybe keeping in in the separate packages + returning interface{} in ReadTokenManagerFromContext() is the best approach.